src/Hydrator/EloquentHydrator.php

```diff
@@ -115,6 +115,32 @@ public function __construct(HttpServiceInterface $service)
     {
         $this->service = $service;
     }
+    
+    /**
+     * @param StandardObjectInterface $attributes
+     * @param $record
+     * @return void
+     */
+    protected function defaultHydrateAttributes(StandardObjectInterface $attributes, $record)
+    {
+        $hydratorAttributes = $this->attributes;
+        if(empty($hydratorAttributes))
+        {
+            $hydratorAttributes = [];
+            foreach($record->getFillable() as $attribute)
+            {
+                if(strpos($attribute, '_') !== false)
```

Comment:
to convert the attribute key you should use the `Str::dasherize` method (the `Str` class already has a `use` statement). There's no need to us `strpos` - just run the attribute key through the `dasherize` method